decimal64.go

```diff
@@ -305,3 +444,21 @@ func (d Decimal64) Sign() int {
 func (d Decimal64) Signbit() bool {
 	return d.bits>>63 == 1
 }
+
+//size returns the magnitude (number of digits) of a uint64.
+func size(n uint64) int {
```

Comment:
nit: Needs a better name. Suggestion: `numDecimalDigits` (shorter would be better, but I can't think of anything off the top of my head).